From 2cb53fed250d4d0fbfed02e5fb0584529fb11b33 Mon Sep 17 00:00:00 2001
From: Marek Vasut <marex@denx.de>
Date: Fri, 14 Feb 2020 09:11:43 +0100
Subject: [PATCH 150/163] FIXME: ARM: dts: stm32: Disable TSC2004 on DHCOM due to IRQ conflict

The TSC2004 and KS8851 share GPIO IRQ on the same EXTI IRQ line,
hence they are in IRQ conflict. Disable the TSC2004 for now.

Upstream-Status: Inappropriate
Signed-off-by: Marek Vasut <marex@denx.de>
---
 arch/arm/boot/dts/stm32mp15xx-dhcom-som.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/stm32mp15xx-dhcom-som.dtsi b/arch/arm/boot/dts/stm32mp15xx-dhcom-som.dtsi
index be647cd9e747..a6e8c5165dd1 100644
--- a/arch/arm/boot/dts/stm32mp15xx-dhcom-som.dtsi
+++ b/arch/arm/boot/dts/stm32mp15xx-dhcom-som.dtsi
@@ -302,6 +302,7 @@
 		reg = <0x49>;
 		vio-supply = <&v3v3>;
 		interrupts-extended = <&gpioh 3 IRQ_TYPE_EDGE_FALLING>;
+		status = "disabled";
 	};
 
 	eeprom@50 {
-- 
2.27.0

